src/Headline.svelte

```diff
@@ -25,6 +25,10 @@
             show_maplink_menu = !show_maplink_menu;
         }
     }
+
+    function substitute_theme_in_url(theme) {
+        return THEME_URL.replace('{theme}', theme);
```

Comment:
You forgot to import THEME_URL.

Reply:
Oops, I forgot one chunk when committing! Should be fixed now.